test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/reliefdetails/ReliefDetailsControllerSpec.scala

```diff
@@ -1379,7 +1379,29 @@ class ReliefDetailsControllerSpec
             )
           }
         }
+        "the user has set residential relief to no" in {
```

Comment:
probably don't need this test - it should be covered by the `forall` in the test above. Also, in this test there would have been no need to do a `forall` since you're populating the  answers with a specific one each time rather than using the generated one

Reply:
I agree with forall, but looks like forall does not test the AmountInPence(0) case. Thanks